src/Kernel32.Tests/Kernel32Facts.cs

```diff
@@ -11,6 +11,7 @@
 using Xunit;
 using static PInvoke.Constants;
 using static PInvoke.Kernel32;
+using System.Collections.Generic;
```

Comment:
For some reason Visual Studio is completely confused with the current state of the solution and see a lot of references as invalid. I had to disable R# as it was flagging a lot of valid code red.

Either I missed the StyleCop warning among all the other warnings or the same problem happens with the analyzer dll.

The fun part is that while the Roslyn instance in VS see unresolvable dependencies the build is completely correct. Maybe I should display build error/warnings only.

----

As a side note the current solution is quite hard to work with in Visual Studio. I seem to have more whole UI freeze (Building, starting unit tests via VS test runner) than normal. I didn't investigate too much if it was something in my config or something in the solution itself (project.json usage ? Code generators ?) it's quite hard to hack on it.